examples/typescript-ex/src/index.ts

```diff
@@ -1,45 +1,48 @@
-import { CloudEvent, HTTPREceiver } from '../../';
+import { CloudEvent, HTTPReceiver } from '../../..';
+import { CloudEventV1 } from '../../../lib/v1';
 
 export function doSomeStuff() {
-  const receiver = new HTTPREceiver();
+  const receiver = new HTTPReceiver();
 
-  const myevent: CloudEvent = new CloudEvent()
-    .source('/source')
-    .type('type')
-    .dataContentType('text/plain')
-    .dataschema('http://d.schema.com/my.json')
-    .subject('cha.json')
-    .data('my-data')
-    .addExtension("my-ext", "0x600");
+  const myevent: CloudEventV1 = new CloudEvent({
+    source: '/source',
+    type: 'type',
+    dataContentType: 'text/plain',
+    dataSchema: 'https://d.schema.com/my.json',
+    subject: 'cha.json',
+    data: 'my-data'
+  });
+  myevent.addExtension('extension-1', 'some extension data');
 
-  console.log(myevent.toString());
-  console.log(myevent.getExtensions());
+  console.log("My structured event:", myevent.toString());
+  console.log("My structured event extensions:", myevent.getExtensions());
 
   // ------ receiver structured
-  const payload = myevent.toString();
   const headers = {
-    "Content-Type":"application/cloudevents+json"
+    "content-type": "application/cloudevents+json"
```

Comment:
It might be worth adding a comment in this example mentioning that header names should be standardized to all lower case.